tox.ini

```diff
@@ -1,15 +1,18 @@
 [tox]
-envlist = py27
+envlist = tests, pylama
```

Comment:
I'm having second thoughts on the environment name. Let's restore `py27`, which I think will also fix the AppVeyor issue. It means we could add a `py36` if we're able to add support for Python 3.